src/main/java/ua/danit/final_project/dto/VacancyDto.java

```diff
@@ -0,0 +1,44 @@
+package ua.danit.final_project.dto;
+
+import lombok.Data;
+import ua.danit.final_project.entities.Vacancy;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+
+@Data
+public class VacancyDto implements Serializable {
+
+  private Long id;
+
+  private String position;
+
+  private String salary;
+
+  private String status;
+
+  private String info;
+
+  private Timestamp publication;
+
+  public VacancyDto(Vacancy vacancy) {
+    this.id = vacancy.getId();
+    this.position = vacancy.getPosition().getTitle();
+    this.salary = vacancy.getSalary();
+    this.status = vacancy.getVacancyStatus().toString();
+    this.info = vacancy.getInfo();
+    this.publication = vacancy.getPublication();
+
+/*    this.date = vacancy.getDate();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=105&issues=AWXJ1tSGlgbYnohY1fRS&open=AWXJ1tSGlgbYnohY1fRS)

[ISSUE_KEY]: AWXJ1tSGlgbYnohY1fRS